CHANGELOG.adoc

```diff
@@ -16,9 +16,14 @@
 |===
```

Comment:
Can you please add a new `## Unreleased` table to start collecting changes for the next release?